package.json

```diff
@@ -31,9 +37,11 @@
   "scripts": {
     "start": "node dist/main.js",
     "build": "tsc --build .tsconfig.json",
-    "test": "jest --no-cache"
+    "test": "jest --no-cache",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix"
   },
   "dependencies": {
+    "eslint-plugin-react": "^7.21.3",
```

Comment:
First of all, thank you for your contribution! ❤️

I have a question here: there is a reason to have a plug-in for react, considering our application is a CLI?

Reply:
Ohh yeah, It was added accidentally. Will remove it.